packages/lesswrong/components/comments/CommentsTimeline.tsx

```diff
@@ -67,16 +67,16 @@ const CommentsTimelineFn = ({
 
   const { CommentWithReplies, Typography } = Components;
 
-  const handleScroll = (e) => {
-    const isAtBottom = Math.abs((e.target.scrollHeight - e.target.scrollTop) - e.target.clientHeight) < 10;
+  const handleScroll = (e: React.UIEvent<HTMLDivElement>) => {
+    const isAtBottom = Math.abs((e.currentTarget.scrollHeight - e.currentTarget.scrollTop) - e.currentTarget.clientHeight) < 10;
 
     // If we are not at the bottom that means the user has scrolled up,
     // in which case never autoscroll to the bottom again
     if (!isAtBottom)
       setUserHasScrolled(true);
 
     // Start loading more when we are less than 1 page from the top
-    if (!loadingMoreComments && commentCount < totalComments && e.target.scrollTop < e.target.clientHeight)
+    if (!loadingMoreComments && commentCount < totalComments && e.currentTarget.scrollTop < e.currentTarget.clientHeight)
```

Comment:
FYI @Will-Howard this is one of the few actual code changes in this PR.  I'm not 100% sure how to test this change; adding the `React.UIEvent<HTMLDivElement>` type to the event caused it to complain that `target` didn't exist, but `currentTarget` does.  According to https://github.com/DefinitelyTyped/DefinitelyTyped/issues/11508#issuecomment-256045682 it seems like we should _probably_ be using currentTarget, not super confident.

Reply:
Ah, this whole file can actually be deleted now. We are now using the `TagSubforumPage2` component which doesn't use this (instead of `TagSubforumPage`), and I don't think there's any chance of going back at this point